frcnn_detector.py

```diff
@@ -61,6 +61,7 @@ def __init__(self, model_path,
         self.__use_selective_search_rois = use_selective_search_rois
         self.__use_grid_rois = use_grid_rois
         self.__model = None
+        self.__isPythonModel = None

```

Comment:
Why None?
If it's a boolean I'd rather initialize it to True/False. 
It's also more efficient